app/controllers/admin/enterprises_controller.rb

```diff
@@ -216,6 +217,14 @@ def collection_actions
       [:index, :for_order_cycle, :visible, :bulk_update]
     end
 
+    def set_enterprise
+      @enterprise = @object
```

Comment:
This instance variable is already set by `ResourceController#load_resource`:
https://github.com/dacook/openfoodnetwork/blob/main/app/controllers/admin/resource_controller.rb#L106-L115

So I would suggest removing that line and then this method perhaps should be named something else like this:

```suggestion
    def require_enterprise
```